src/test/java/core/basesyntax/strategy/PurchaseActionTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.actions.BalanceActionHandler;
+import core.basesyntax.strategy.actions.PurchaseActionHandler;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PurchaseActionTest {
+    @Test
+    public void purchaseAction_ok() {
+        Storage.getFruits().clear();
```

Comment:
read about `@BeforeEach`

Reply:
still not fixed. As @petro-ushchuk advised, let's clear storage in a separate method